components/meter/meter-mixin.js

```diff
@@ -21,6 +21,11 @@ export const MeterMixin = superclass => class extends LocalizeCoreElement(superc
 			 * @type {string}
 			 */
 			text: { type: String },
+			/**
+			 * Displays the text as a label beneath the meter
```

Comment:
Actually, the inverse? You'll probably want to adjust the wording for `text` to be "Displayed beneath the meter. If the text contains..." and then this one can be something like `Hides the text`.